### PR TITLE
Simplifying using rasa

### PR DESCRIPTION
**Pull request by beeva-fernandocerezal**
**Proposed changes**:
- Changed Dockerfile for enabling a model just after container is built
- Changed entrypoint for setting an unique existing model as default model (rasa look for a default model, but it is not set after training)
- Added script for managing rasa container. It avoids people needs to know docker in order to use rasa. Rasa needs to be rebooted in order to load previous trained models. This script makes this easier for people don't know the difference between docker build, run, start and stop.

After those changes you have to execute "./manage_container build", send the training json using curl and execute "./manage_container restart" for getting a working rasa.

### CODE REVIEW

**Comment by twerkmeister** [comment]
Thanks for the submission! Two things would make this really helpful for others: 1. printing usage information when a user doesn't input the right keywords into the manage script and 2nd some updates to the readme.md in how to use the script :+1:

**Comment by beeva-fernandocerezal** [comment]
Hello, I added the message in the manage script and added documentation to readme about how to use the proposed changes

File: README.md [hunk 1]
```diff
@@ -43,23 +43,57 @@ curl 'http://localhost:5000/parse?q=hello'
...
 
+#### B.2 Install with Docker automatically
+Automatically installation uses spacy english model and configure trained model as default if there is only one trained model.
+If you want to use another model read the Dockerfile, comment spacy english model lines and uncomment the lines for your selected model.
+
+##### 1. Create container
+
+Execute as root:
+```
+chmod +x ./manage_container
```
Comment (twerkmeister) [inline README.md]: this is something you can already commit to the repository

Comment (beeva-fernandocerezal) [inline README.md]: I'm sorry, I don't understand. What do you mean?

Comment (tmbo) [inline README.md]: He is talking about the `manage_container` file you added. You can execute can execute `chmod +x ./manage_container` locally and comit the file with the changed permissions to github (github should persist these permissions as well) - hence, you can remove this line after you changed and added the file with the updated permissions to github.

**Comment by beeva-fernandocerezal** [comment]
I removed the line in the readme. The script already has execution permissions. I thought it was file system data and and git didn't take care about it.

File: README.md [hunk 2]
```diff
@@ -43,23 +43,57 @@ curl 'http://localhost:5000/parse?q=hello'
...
 curl 'http://localhost:5000/parse?q=hello'
 ```
 
+#### B.2 Install with Docker automatically
+Automatically installation uses spacy english model and configure trained model as default if there is only one trained model.
+If you want to use another model read the Dockerfile, comment spacy english model lines and uncomment the lines for your selected model.
+
+##### 1. Create container
+
+Execute as root:
```
Comment (twerkmeister) [inline README.md]: You could also state both things as options

**Comment by tmbo** [comment]
@beeva-fernandocerezal do you mind integrating the feedback? It would be awesome if we can get your improvements in the hands of others as soon as possible 😃

**Comment by tmbo** [comment]
@beeva-fernandocerezal I don't want this to get lost, mind having a look at the feedback from thomas?

**Comment by beeva-fernandocerezal** [comment]
Maybe next week, but I need to review what is useless or wrong with new version.

**Comment by tmbo** [comment]
Sounds good :+1: let me know if you need any help.

**Comment by tmbo** [comment]
Closed due to inactivity. Let me know if you still want to get this merged.

**Pull request closed without merging**